src/main/java/core/basesyntax/operation/ReturnOperation.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.dao.FruitShopService;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private final FruitShopService fruitShopService;
+
+    public ReturnOperation(FruitShopService fruitShopDao) {

```

Comment:
```suggestion
    public ReturnOperation(FruitShopService fruitShopService) {
```
the same for other handlers